src/modules/editor/sagas.ts

```diff
@@ -541,3 +588,52 @@ function handleEntitiesOutOfBoundaries(args: { entities: string[] }) {
     store.dispatch(setEntitiesOutOfBoundaries(entities))
   }
 }
+
+function* getWearables(items: Item[]) {
+  const bodyShape: WearableBodyShape = yield select(getBodyShape)
+  const avatar = (bodyShape === WearableBodyShape.MALE ? maleAvatar : femaleAvatar) as Wearable[]
+  const apply = items.map(item => toWearable(item))
+  const wearables = mergeWearables(avatar, apply)
+  return wearables
+}
+
+function* renderAvatar() {
+  if (yield select(isReady)) {
+    const visibleItems: Item[] = yield select(getVisibleItems)
+    const wearables = yield getWearables(visibleItems)
+    const animation = yield select(getAvatarAnimation)
+    yield call(async () => {
+      console.log('render', wearables, animation)
+      editorWindow.editor.addWearablesToCatalog(wearables)
+      editorWindow.editor.sendExternalAction(updateAvatar(wearables, animation))
+    })
+  }
+}
+
+function* bustCache() {
+  const wearables = yield getWearables([])
```

Comment:
Maybe this is a non-problem but I find the method hard to understand. I thought it got all wearables but I don't really get what's doing when we just send `[]`.
Could just be me but maybe we can find a better name for it?

Reply:
yeah that was actually doing to things (converting`Item`s into `Wearable`s and merging with the default wearables. I split it into separate functions to make it clearer.